src/components/core/DpDataTable/DpSelectPageItemCount.vue

```diff
@@ -1,9 +1,28 @@
+<template>
+  <div>
+    <select
+      id="item-count"
```

Comment:
using a hardcoded id in a component that may have multiple instances within a view will lead to invalid html. this may be solved with a [composable like this one](https://github.com/demos-europe/diplanportal-frontend/blob/main/composables/util/useUniqueComponentId.ts).